### PR TITLE
Add blog post for 2.30.0

### PR DESCRIPTION
**Pull request by eheimbuch**
Blogpost for 2.30.0

### CODE REVIEW

**Comment by pfeuffer** [comment]
I would use only light mode screenshots

File: content/posts/2022-01-21-scm-manager-2-30-0/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## Alerting
+We have already been working on an alert system that will warn you directly in your instance when an instance to warn you when a vulnerability is found that affects your installed SCM-Manager core and plugin versions.
+To make sure you don't miss any important patches, each user is notified of existing security vulnerabilities.
+These alerts in the header will only appear if there are known vulnerabilities that affect your installed versions.
+
+![Alerts](assets/alerting.png)
+
+## Compare
+Another code related feature we have added is 'Compare'. 
+You can now compare diffs and changesets between two branches, tags or revisions.
```
Comment (pfeuffer) [inline content/posts/2022-01-21-scm-manager-2-30-0/index.md:29]: "Compare diffs" sounds strange. "Create diffs" would be better, but this does not fit to the following changesets 🤔

**Review by pfeuffer — COMMENTED** [review]
Like this one. I only stumpbled upon the "compared diff".

File: content/posts/2022-01-21-scm-manager-2-30-0/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: SCM-Manager 2.30.0
+date: 2022-01-21T17:00:00+0000
```
Comment (eheimbuch) [inline content/posts/2022-01-21-scm-manager-2-30-0/index.md:3]:
```suggestion
date: 2022-01-24T17:00:00+0000
```

File: content/posts/2022-01-21-scm-manager-2-30-0/index.md (hunk 1 above)
Comment (eheimbuch) [inline content/posts/2022-01-21-scm-manager-2-30-0/index.md:29]:
```suggestion
You can now create diffs and list changesets between two branches, tags or revisions. 
```

**Pull request merged**